mix/mix_protocol.nim

```diff
@@ -127,6 +155,20 @@ proc handleMixNodeConnection(
     trace "Exit node - Received mix message",
       receiver = multiAddr, message = deserialized.message, codec = deserialized.codec
 
+    when defined(enable_mix_benchmarks):
```

Comment:
Optional: If we plan to expand benchmark support or keep these logs around, it might be worth extracting the repeated logging logic in `handleMixNodeConnection` (_e.g.,_ `startTime`, `endTime`, `procDelay`, shared log fields) into a small helper.
This would complement the existing `SendPacketConfig` abstraction used in `sendPacket` and help reduce duplication across `Exit`, `Reply`, _etc._ paths.